editor/src/uuiui/styles/css-vars.tsx

```diff
@@ -0,0 +1,18 @@
+import { useColorThemeVariables } from './theme'
+import React from 'react'
+
+export const ColorThemeStyleComponent = React.memo(() => {
+  const colorTheme = useColorThemeVariables(),
+    { inverted, ...currentTheme } = colorTheme
+
+  return (
+    <style>
+      {':root {'}
+      {Object.entries({ ...inverted, ...currentTheme }).map((entry) => {
+        const [variable, value] = entry
```

Comment:
You can deconstruct the entry directly in the inlined function's params like so
```suggestion
      {Object.entries({ ...inverted, ...currentTheme }).map(([variable, value]) => {
```

Reply:
I was getting type issues when I tried to do this. I'll try again

Reply:
worked this time

Reply:
Perhaps you did `.map([k,v] =>` which would have failed, as the parens are required for the deconstruction to work